src/main/kotlin/no/skatteetaten/aurora/boober/service/resourceprovisioning/ExternalResourceProvisioner.kt

```diff
@@ -45,7 +46,7 @@ class ExternalResourceProvisioner(
                     it.key.toString() to it.value.toString().toByteArray()
                 }
             }?.let {
-                VaultSecretEnvResult(secret.name, it.toMap())
+                VaultSecretEnvResult(secret.name.ensureStartWith(deploymentSpecInternal.name, "-"), it.toMap())
```

Comment:
Jeg gjør dette her for å unngå å gjøre det 2 plasser senere. Siden både secret name og name i secretKeyRef må være samme.